README.md

```diff
@@ -94,13 +93,9 @@ The online documentation is available at https://opentelemetry-python.readthedoc
 if you want to access the documentation for the latest version use
 https://opentelemetry-python.readthedocs.io/en/latest/.
 
-## Compatible Exporters
-
-See the [OpenTelemetry registry](https://opentelemetry.io/registry/?s=python) for a list of exporters available.
-
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
```

Comment:
I'd argue that "contributing documentation" doesn't provide much more semantic value, while the filename helps embed the location in people's minds. but minor nitpick.

Reply:
Fair enough, I've updated it back to CONTRIBUTING.md